takeshape/takeshape-starter-ecommerce-kitchen-sink
# Adds product pages

Adds dedicated product pages.

As a bonus, adds Typescript config and generates types from TakeShape schema. [I love the smell of dogfood in the morning.](https://www.youtube.com/watch?v=nx5N-4JvVyk)

<img width="1272" alt="Screen Shot 2022-03-25 at 5 44 34 PM" src="https://user-images.githubusercontent.com/6089482/160206327-e0c07831-92c1-41f0-95cc-0ab6ccff1fe4.png">

### Test Steps

1. Checkout branch and run `npm i && npm run dev`
2. Clicking on any product image or name should take you to its dedicated page.
